### PR TITLE
KoalaBear84.OpenDirectoryDownloader version 2.8.0.1

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/106261)

### CODE REVIEW

**Comment by upintheairsheep** [comment]
This package failed to install with:

ShellExecute installer failed: 2147516547

File: manifests/k/KoalaBear84/OpenDirectoryDownloader/2.8.0.1/KoalaBear84.OpenDirectoryDownloader.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: KoalaBear84.OpenDirectoryDownloader
+PackageVersion: 2.8.0.1
+Installers:
+- InstallerUrl: https://github.com/KoalaBear84/OpenDirectoryDownloader/releases/download/v2.8.0.1/OpenDirectoryDownloader-2.8.0.1-win-x64.zip
+  Architecture: x64
+  InstallerType: zip
+  NestedInstallerType: exe
```
Comment (Trenly) [inline manifests/k/KoalaBear84/OpenDirectoryDownloader/2.8.0.1/KoalaBear84.OpenDirectoryDownloader.installer.yaml:10]:
```suggestion
  NestedInstallerType: portable
```

**Review by Trenly — CHANGES_REQUESTED** [review]
Has a dependency on the .NET Runtime
```
App: C:\Users\WDAGUtilityAccount\AppData\Local\Temp\WinGet\KoalaBear84.OpenDirectoryDownloader.2.8.0.1\extracted\OpenDirectoryDownloader.exe
Architecture: x64
App host version: 7.0.2
.NET location: Not found

Learn about runtime installation:
https://aka.ms/dotnet/app-launch-failed

Download the .NET runtime:
https://aka.ms/dotnet-core-applaunch?missing_runtime=true&arch=x64&rid=win10-x64&apphost_version=7.0.2
PS C:\Users\WDAGUtilityAccount\AppData\Local\Temp\WinGet\KoalaBear84.OpenDirectoryDownloader.2.8.0.1\extracted> $LASTEXITCODE
-2147450749
```

File: manifests/k/KoalaBear84/OpenDirectoryDownloader/2.8.0.1/KoalaBear84.OpenDirectoryDownloader.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: KoalaBear84.OpenDirectoryDownloader
+PackageVersion: 2.8.0.1
```
Comment (stephengillie) [inline manifests/k/KoalaBear84/OpenDirectoryDownloader/2.8.0.1/KoalaBear84.OpenDirectoryDownloader.installer.yaml:5]:
```suggestion
PackageVersion: 2.8.0.1
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.DotNet.DesktopRuntime.7
```

**Comment by ItzLevvie** [comment]
[Policy] Blocking-Issue

**Review by stephengillie — APPROVED** [review]

**Pull request merged**